pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -811,6 +1295,53 @@ def read_mwa_corr_fits(
                         flags,
                     )
 
+            if flag_init:
+                self.flag_init(
+                    num_fine_chans,
+                    edge_width=edge_width,
+                    start_flag=start_flag,
+                    end_flag=end_flag,
+                    flag_dc_offset=flag_dc_offset,
+                )
+
+            # to account for discrepancies between file conventions, in order
+            # to be consistent with the uvw vector direction, all the data must
+            # be conjugated
+            self.data_array = np.conj(self.data_array)
+
+        # wait for LSTs if set in background
+        if proc is not None:
+            proc.join()
+
+        if not self.metadata_only:
+            # reorder polarizations
+            # reorder pols calls check so must come after
+            # lst thread is re-joined.
+            self.reorder_pols()
```

Comment:
You could call this with `run_check=False` if you want to since you're calling check at the end of this. Might save a little time.